### DISCUSSION TITLE
Custom log ingestion

### DISCUSSION THREAD

**Discussion by ongyo-sensei** [matanolabs/matano#160] [Q&A] [answered]
Hello,

What would be best way to import custom audit logs (ont supported/managed) to Matano?

From what I understand I need to pull data in custom S3 bucket on my own and then normalize to Elastic Common Schema and  and then use Log configuration to ingest it as described [here](https://www.matano.dev/docs/log-sources/configuration) in documentation.

Is that roughly the process?

I just need to understand the process.

Thank you!

**Comment by Samrose-Ahmed** [comment, marked as answer]
Yes thats the process, put the raw logs into S3, define your schema, and transformation script in a custom log source and consume your data from the data lake.

**Reply by ongyo-sensei** [reply]
Thank you!
